capitoli/Quinto_Passo_Primi_Programmi.rst

```diff
@@ -49,53 +51,38 @@ Le due scatole conterranno il numero 5.
-| PLUTO=15                 |                         |
-|                          |                         |
-| print("PIPPO=",PIPPO)    | PIPPO=5                 |
-|                          |                         |
-| print("PLUTO=",PLUTO)    | PLUTO=15                |
-|                          |                         |
-| PIPPO=PLUTO              |                         |
-|                          |                         |
-| print("Dopo l'esecuzione | Dopo l'istruzione       |
-| dell'istruzione          |                         |
-|                          | PLUTO=15                |
-| PIPPO=",PIPPO)           |                         |
-|                          |                         |
-| print("Dopo l'esecuzione | PIPPO=15                |                  
-| dell'istruzione          |                         |
-| PLUTO=",PLUTO)           |                         |         
-|                          |                         |
-+--------------------------+-------------------------+
-|**Programma2 PLUTO=PIPPO**| **Il risultato sarà:**  |
-+--------------------------+-------------------------+
-| PIPPO=5                  |                         |
-|                          |                         |
-| PLUTO=15                 |                         |
-|                          |                         |
-| print("PIPPO=",PIPPO)    | PIPPO=5                 |
-|                          |                         |
-| print("PLUTO=",PLUTO)    | PLUTO=15                |
-|                          |                         |
-| PLUTO=PIPPO              |                         |
-|                          |                         |
-| print("Dopo l'esecuzione | Dopo l'istruzione       |
-| dell'istruzione          |                         |
-|                          | PLUTO=5                 |
-| PLUTO=",PLUTO)           |                         |
-|                          |                         |
-| print("Dopo l'esecuzione | PIPPO=5                 |                  
-| dell'istruzione          |                         |
-| PIPPO=",PIPPO)           |                         |         
-|                          |                         |
-+--------------------------+-------------------------+
-
-**II istruzione**::boltblue:`INCREMENTO` 
+**Programma1 PIPPO=PLUTO**
+::
+
+	PIPPO=5
+	PLUTO=15
+	print ("PIPPO = ", PIPPO)
+	print ("PLUTO = ", PLUTO)
+	PIPPO = PLUTO
+	print ("Dopo l'esecuzione dell'istruzione PIPPO = ", PIPPO) 
+	print ("Dopo l'esecuzione dell'istruzione PLUTO = ", PLUTO)
+	
+	# Risultato Atteso:
+		# PIPPO = 5
+		# PLUTO = 15
+		# Dopo l’esecuzione dell’istruzione PLUTO = 15 
+		# PIPPO = 15
+
+**Programma 2 PLUTO = PIPPO**
+
+.. activecode:: Copiatura
+   :coach:
+   :caption: Come copiare
+   
+   PIPPO=5
+   PLUTO=15
+   print ("PIPPO = ", PIPPO)
+   print ("PLUTO = ", PLUTO)
+   print ("PLUTO =", PIPPO) 
+   print ("PIPPO = ", PIPPO) 
+
+
+:boltblue:`INCREMENTO` 
```

Comment:
Qui metterei proprio un titolo con `----` in modo tale da farlo apparire nel TOC dell'index